frontend/src/lib/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -135,20 +141,26 @@ class Selectbox extends React.PureComponent<Props, State> {
 
   public render(): React.ReactNode {
     const style = { width: this.props.width }
-    const { label, labelVisibility, help } = this.props
+    const { label, labelVisibility, help, clearable, theme } = this.props
     let { disabled, options } = this.props
 
+    const noOptionsMsg = "No options to select."
+    const chooseOptionMsg = "Choose an option"
     let value = [
       {
         label:
-          options.length > 0
+          options.length > 0 && this.state.value
             ? options[this.state.value]
-            : "No options to select.",
-        value: this.state.value.toString(),
+            : clearable
+            ? null
+            : noOptionsMsg,
+        value: this.state.value
```

Comment:
Same here. Don't we need to check for `None` or `undefined` otherwise this won't work with `0`?

Reply:
True, have changed it 👍